### ISSUE TITLE
Unable to run "python -m sxample_scenes [Example] -pl"

### ISSUE THREAD

**Issue by KaizenHayashi** [3b1b/manim#1589]
![Screenshot 2021-08-01 010643](https://user-images.githubusercontent.com/86646718/127747348-ab7d2c27-d8e3-4534-987d-c1883d120cbf.jpg)

I followed the video tutorial on this website here step by step:
[https://financial-engineering.medium.com/manim-how-to-install-manim-on-windows-10-f939f5418d79](url)
I may have installed the required files in different locations, but I tried linking them all to PATH under the Environment Variables settings.

The example doesn't show up in any video players and the cmd immediately moves on and asks for a new input.
I am completely new to Python, so I would not know just about everything. I downloaded Manim with the intention to animate Mathematics, but of course willing to learn Python along the way.

As of now, my current guess of how Manim would work is that I code somewhere on an app like Visual Studio, extract scenes and then run it in cmd?
I most definitely still need a lot of guidance on how to use it.

I also do not know what kind of info should I provide here in order to solve my issue. So it might be a thread that will go on for some time, but I would appreciate all the help I can get.

Thank you.

**Comment by TonyCrane** [comment]
please follow the tutorial in the README file.

If you are using the version in `cairo-backend` branch, you can just download the code and run:
```sh
python -m manim example_scenes.py SquareToCircle -pl
# or: python manim.py example_scenes.py SquareToCircle -pl
```
notice that it is **not** `python -m example_scenes ...`

If you want to use the newest version of 3b1b/manim, please follow the official tutorial step by step: https://3b1b.github.io/manim/getting_started/installation.html

**Comment by KaizenHayashi** [comment]
I see the tutorial now, thanks.
Do I need to uninstall the non Manim files from the video in the webpage I linked to?
The last time I tried to delete MiKTeX using cmd because I thought that it must be in :C drive and not AppData, it kept failing to delete and I had to use `del /s` to remove the subfolders one by one until it is disrupted enough to be deleted traditionally.

I also downloaded the .zip file from the "Code" section of this GitHub project, and I looked through the Branches and it mentioned nothing of "cairo-backend", do you mind clafirying what that is and where I can find and check it?

Also when I tried to run `python -m manim example_scenes.py SquareToCircle -pl` it says: `No module named manim`
So I deleted the "manim" part and it responded differently, so I thought the situation improved. Is there anything wrong with what I did?

I really hope that I can follow the tutorial stated in the README.md file without having to undo whatever I have done too much.
Thank you once again.

**Comment by TonyCrane** [comment]
> Do I need to uninstall the non Manim files from the video in the webpage I linked to?

No, you don't have to. Just remove the manim folder and download the newest one from github is ok.

If you want to use the newest version of 3b1b/manim, just follow the tutorial completely should be ok. And notice that you should `pip install -e .` first, and then use `manimgl example_scenes.py OpeningManimExample`.
Besides, you can also run the newest version without downloading the code from here, just `pip install manimgl` and then run `manimgl example_scenes.py OpeningManimExample` is also ok.

And if you want to use the old version of manim(cairo-backend), the code is here: https://github.com/3b1b/manim/tree/cairo-backend
You can just download it and run `python -m manim example_scenes.py SquareToCircle -pl` in the folder.

**Comment by KaizenHayashi** [comment]
![Screenshot 2021-08-01 164717](https://user-images.githubusercontent.com/86646718/127765108-02762b3f-d5a2-4f03-97ff-45b41f8649b4.jpg)
I did `pip install -e .` and this happened. Would a newer version of pip affect anything negatively?

**Comment by KaizenHayashi** [comment]
![Screenshot 2021-08-01 164717](https://user-images.githubusercontent.com/86646718/127765158-397354a5-17f9-4c55-864b-679370bc596f.jpg)

I did `pip install -e .` and this showed up. Would a newer version of pip affect anything negatively?

**Comment by TonyCrane** [comment]
Maybe you need admin privilege to open a new cmd and run it again.

**Comment by KaizenHayashi** [comment]
May I gets the steps to do that? Who is the one in need of the permission?

Or at the very least, the most I know of is giving folders and documents permissions through Properties.
